### PR TITLE
PXB-3234 xbstream documentation missing options (including decompress…

### PR DESCRIPTION
**Pull request by patrickbirch**
…) (8.0)

 On branch pxb-3234-8.0
	modified:   docs/xbcloud-options.md
	modified:   docs/xbcloud-swift.md

### CODE REVIEW

File: docs/xbcloud-options.md [hunk 1]
```diff
@@ -1,60 +1,297 @@
 # The xbcloud command-line options
 
-The xbcloud binary has the following command line options:
+Usage: 
+ 
+ ```{.bash data-prompt=text}
```
Comment (alina-derkach-oaza) [inline docs/xbcloud-options.md]:
```suggestion
```{.bash data-prompt=$}
```

File: docs/xbstream-options.md [hunk 2]
```diff
@@ -1,43 +1,102 @@
 # The xbstream command-line options
 
+ ```{.bash data-prompt=text}
```
Comment (alina-derkach-oaza) [inline docs/xbstream-options.md]:
```suggestion
```{.bash data-prompt=$
```

Comment (alina-derkach-oaza) [reply docs/xbstream-options.md]: The code block is broken

File: docs/xbcloud-options.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/xbcloud-options.md]: The code block is broken

Comment (patrickbirch) [reply docs/xbcloud-options.md]: pseudo-code
```{.text .no-copy}

File: docs/xbcloud-options.md [hunk 3]
```diff
@@ -1,60 +1,297 @@
 # The xbcloud command-line options
 
-The xbcloud binary has the following command line options:
+Usage: 
+ 
+ ``````{.text .no-copy}
```
Comment (alina-derkach-oaza) [inline docs/xbcloud-options.md]:
```suggestion
 ```{.text .no-copy}
```

File: docs/xbstream-options.md [hunk 4]
```diff
@@ -1,43 +1,102 @@
 # The xbstream command-line options
 
+ ``````{.text .no-copy}
```
Comment (alina-derkach-oaza) [inline docs/xbstream-options.md]:
```suggestion
 ```{.text .no-copy}
```

**Review by alina-derkach-oaza — APPROVED** [review]

**Pull request merged**